solutions/web3-fetching-strategies/pages/index.tsx

```diff
@@ -0,0 +1,88 @@
+import Head from 'next/head'
+import { Layout, Text, Page, Code, Link } from '@vercel/examples-ui'
+import { BAYCAbi__factory } from '../types/ethers-contracts'
+import { ethers } from 'ethers'
+import useSWR from 'swr'
+
+const contract = BAYCAbi__factory.connect(
+  '0xBC4CA0EdA7647A8aB7C2061c2E118A18a936f13D',
```

Comment:
This is public?

Reply:
Yep! It's the Smart Contract of Bored Ape Yatch club.

Reply:
@gbibeaul Can you add it to a const called `BORED_API_YATCH_CLUB_ADDRESS` or similar

Reply:
Done!